Header header
geometry_msgs/Pose pose
geometry_msgs/Vector3 dimensions #x, y and z
# You can use this field to hold value such as likelihood
float32 value
uint32 label